connect/runtime/src/test/java/org/apache/kafka/connect/runtime/AbstractHerderTest.java

```diff
@@ -261,6 +262,9 @@ private AbstractHerder createConfigValidationHerder(Class<? extends Connector> c
         EasyMock.expect(herder.generation()).andStubReturn(generation);
 
         // Call to validateConnectorConfig
+        EasyMock.expect(worker.configTransformer()).andReturn(transformer).times(2);
```

Comment:
why not once, but twice?

Reply:
Because of the two calls in 
```
         if (worker.configTransformer() != null) {
             connectorProps = worker.configTransformer().transform(connectorProps);
         }
```